src/html/index.html

```diff
@@ -4,21 +4,38 @@
         <meta charset="UTF-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>Gulp</title>
+        <title>API</title>
         <link rel="stylesheet" href="../css/main.css">
     </head>
 
-    <body class="overlay">
-        @@include("./page-header/index.html")
-
+    <body>
         <main>
-            <div class="modal hide" id="modal-welcome">
-                <h2>Welcome</h2>
-                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Atque facere nulla rem mollitia expedita eum voluptate quod asperiores quasi animi consequuntur, repellendus eaque pariatur laborum, nemo sed? Quam, voluptatem beatae.</p>
-                <button>Close</button>
+            <div class="wrapper">
+                <form id="main-form">
```

Comment:
w sytuacjach gdy dane z inputa nie sa nigdzie wysyłane do bazy, to nie potrzebujesz całego formularza tworzyć, mogłas uzyc samego elementu <input>, ale spoko że udalo sie tez z formem